samples/composition/src/main/java/com/instacart/formula/samples/composition/ItemPageRenderModel.kt

```diff
@@ -0,0 +1,5 @@
+package com.instacart.formula.samples.composition
+
+import com.instacart.formula.samples.composition.item.ItemRenderModel
+
+data class ItemPageRenderModel(val items: List<ItemRenderModel>)
```

Comment:
It would be cool to have a `toolbarTitle` here or something else too, to drive home the fact that the `RenderModel` renders the entire page

Reply:
Sounds good, will add it.